src/main/java/org/apache/commons/crypto/random/OpenSslCryptoRandom.java

```diff
@@ -44,46 +43,25 @@
 class OpenSslCryptoRandom extends Random implements CryptoRandom {
     private static final long serialVersionUID = -7828193502768789584L;
 
-    /** If native SecureRandom unavailable, use java SecureRandom */
-    private final JavaCryptoRandom fallback;
-    private static final boolean nativeEnabled;
-
     static {
-        boolean opensslLoaded = false;
         if (Crypto.isNativeCodeLoaded()) {
             try {
                 OpenSslCryptoRandomNative.initSR();
-                opensslLoaded = true;
             } catch (Exception t) {// NOPMD
             }
```

Comment:
What if the native code is not loaded, or initSR fails?
This should surely be noted here in a final boolean and used in the ctor and methods?